lambdas/generator/handler.py

```diff
@@ -130,24 +113,11 @@ def get_important_file_names(github_username, repository_name, languages):
     return important_file_names
 
 
-def get_repository_names(github_username):
-    # GitHub API
-    url = "https://api.github.com/users/" + github_username + "/repos"
-    res = requests.get(url, headers={'Authorization-Type': "Bearer " + os.environ['GITHUB_TOKEN'], "X-GitHub-Api-Version": "2022-11-28"})
+def get_important_file_paths(github_username, repo_name, important_file_names):
```

Comment:
need a better name. what does 'important' refer to?
you can either add a description here or rename it so its more descriptive